### ISSUE TITLE
rclone rcd --addr unknown flag

### ISSUE THREAD

**Issue by kouhe3** [rclone/rclone#6675]
rclone version
rclone 1.61.1-termux
- os/version: unknown
- os/kernel: 4.19.157-perf+ (aarch64)
- os/type: android
- os/arch: arm64
- go/version: go1.19.4
- go/linking: dynamic
- go/tags: noselfupdate

rclone rcd --addr :5527
Error: unknown flag: --addr
Usage:
  rclone rcd <path to files to serve>* [flags]

Flags:
  -h, --help   help for rcd

Use "rclone [command] --help" for more information about a command.
Use "rclone help flags" for to see the global flags.
Use "rclone help backends" for a list of supported services.

2023/01/08 06:06:33 Fatal error: unknown flag: --addr

**Comment by ncw** [comment]
I think this is a bug in the docs... You should be using `--rc-addr` I think.

**Comment by kouhe3** [comment]
> I think this is a bug in the docs... You should be using `--rc-addr` I think.

yes. you are right.  --rc-addr , --rc-key, --rc-cert also works fine.

We need to modify docs.

**Comment by Zacho2** [comment]
From a look through the code, I see two issues:

1. Command definitions don't include the prefix in the doc strings. Need to add a way to inject the prefix into the doc strings.
2. Command prefixes are added after the command doc strings are generated. Need to ensure prefixes/docs are handled before adding all the subcommands to the root command (I see some init/global var ordering happening).

I'll work on this issue if needed.

**Comment by ncw** [comment]
Yes you are right. I think changing `libhttp.Help libhttp.TemplateHelp  libhttp.AuthHelp` into functions which take a prefix parameter would work and fix 1. and 2.

Do you want to have a go at this @Zacho2 ?
